__tests__/endpoints.test.js

```diff
@@ -73,4 +74,33 @@ describe('Test Endpoints', () => {
             })
         })       
     });
+    describe('/api/articles', () => {
+        test('200 - responds with articles array', () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
```

Comment:
It'd be better to check the length of the array to make sure enough articles come back from the server